phala-cloud/confidential-ai/confidential-model/confidential-ai-api.mdx

```diff
@@ -1,42 +1,45 @@
 ---
-description: Secure LLM inference API running in GPU Trusted Execution Environment
-  (TEE) for confidential AI computing.
-title: GPU TEE Inference API
+description: Secure LLM inference API running in GPU TEE for confidential AI computing.
+title: Confidential AI API
 ---
 
 
-Phala Cloud provides LLM inference API by connecting to Redpill. By navigating to **Dashboard**->**GPU TEE API** page, you will see details.
+Phala Cloud provides LLM inference API by connecting to Redpill. By navigating to **Dashboard -> Confidential AI API** page, you will see details.
```

Comment:
We should include a basic description of the benefits of using API (the why paragraph).

Reply:
The content in this page is pretty long. It's good to have a short overview section at the top